plugin/api.go

```diff
@@ -174,6 +174,9 @@ type API interface {
 	// GetTeamMember returns a specific membership.
 	GetTeamMember(teamId, userId string) (*model.TeamMember, *model.AppError)
 
+	// GetTeamMembersForUser returns team members for all teams a user is on
```

Comment:
Can you please document the minimum server version here?

```suggestion
	// GetTeamMembersForUser returns all team memberships for a user.
	// Minimum server version: 5.6
```